lib/chf/import/credit_builder.rb

```diff
@@ -1,33 +0,0 @@
-module Chf
```

Comment:
what's with this going away? Related to update, or just something we weren't using anymore anyway? Same for DateOfWorkBuilder.